documentation/en/default-lotus-provider-config.toml

```diff
@@ -25,6 +25,18 @@
   # type: int
   #WinningPostMaxTasks = 0
 
+  # EnableParkPiece enables the "piece parking" task to run on this node. This task is responsible for fetching
+  # pieces from the network and storing them in the storage subsystem until sectors are sealed. This task is
+  # only applicable when integrating with boost, and should be enabled on nodes which will hold deal data
+  # from boost until sectors containing the related pieces have the TreeD/TreeR constructed.
+  # Note that future Curio implementations will have a separate task type for fetching pieces from the internet.
+  #
+  # type: bool
+  #EnableParkPiece = false
```

Comment:
Should we add it to the "storage" default layer?

Reply:
Probably not storage.

Before we move on from boost this will be somewhat awkward, basically this data should live on a node next to boost (so that data copy cost is minimized) or on a node next to/on nodes which execute SDRTrees. Also possible to put this buffer on a separate node type, possibly with cheaper-per-churn-gb-storage, but given how not-scalable boost is anyways that probably doesn't really make sense.

In the future when we integrate the data fetching parts of boost into Curio, most piece-related activities will just happen on the nodes which handle deal downloads+verification.

If we want a simple default that "just works" I'd either put this on configs which enable SDRTrees, or, if we decide to move the boost api adapter (lotus-shed/lpdeal) to core curio (which we might want to do, I'm still thinking through the best approach here), one would just also run the ParkPiece on that node.

(In the future-future when we implement unsealed copy sharing, retrievals probably will become primarly served from PiecePark, then it will make sense to shift the default to enabling this task on storage nodes, but at the same time this specific task will probably become less commonly used in setups, as we'll no longer rely on boost, and curio-native market will have different ways of getting pieces into the system)

Reply:
👍

Reply:
By concern is creeping-in user-complexity. I would like a very small number of new configurations that the SPs need to be aware of. That gets people started faster and improves engagement as we grow the technologies. 
I think it's understandable that it's not perfectly optimal while it's new. 
I recommended 'storage' because this is a low-CPU, higher-reliability-needed operation.

Reply:
Yeah, putting in storage would make this simpler, but this tasks is pretty high bandwidth